packages/nimble-components/src/utilities/tests/setup.ts

```diff
@@ -10,3 +10,4 @@ require('./setup-configuration.js');
 
 // all browser test scripts
 importAll(require.context('../../', true, /\.spec\.js$/));
+importAll(require.context('../../../../build', true, /\.spec\.js$/));
```

Comment:
Instead of the whole build dir can you point to this specific context:

```suggestion
importAll(require.context('../../../../build/generate-workers/dist/esm', true, /\.spec\.js$/));
```

That way both these paths are search dist/esm directories.